app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command, copied_file, file_to_copy = command.split()
```

Comment:
validate that command.split() will have exactly 3 params

Reply:
Validate in code? I understand that according to the conditions of the task there will be only 3 parameters, or I'm mistaken? "Function take only one argument command, that is string with **command cp**, **file name to copy** and **new file name**, separated by spaces."

Reply:
yes, but if command is wrong or has only one argument, like command = "cp txt.txt" or = "file.txt" function will broke. You can validate this : if len(command.split()) == 3

Reply:
change order